README.md

```diff
@@ -8,21 +8,25 @@ Repositori ini berisi kurikulum yang sengaja dibentuk oleh MALANG JS, materi-mat
 
 Materi ini berisi tentang prosedur yang benar dalam menggunakan Node.js. Mulai dari cara install sampai cara membuat proyek baru berbasis Node.
 
+Pengenalan NPM packages dan ekosistemnya.
+
+Pengenalan ES6 yang optional untuk kemudahan dalam programming di NodeJS
+
 ### [S01E02] API Dasar Node.js
```

Comment:
Harusnya ada materi basic js di awal2 session.
Salah satu core mater di nodeschool itu justru basic javascript, contoh javascripting. javascripting workshopper ini materi basic [javascript](https://github.com/sethvincent/javascripting). Dan harus mulai dari ES5 dulu, jangan langsung membandingkan ES5 vs ES6 karena goal penguasaan materinya gak kecapai, yg ada malah peserta bingung.

Pls refer this link: [Workshopper list](http://nodeschool.io/#workshopper-list)